drivers/cc2420/cc2420.c

```diff
@@ -200,14 +201,18 @@ int cc2420_rx(cc2420_t *dev, uint8_t *buf, size_t max_len, void *info)
         uint8_t rssi;
         cc2420_fifo_read(dev, &rssi, 1);
         DEBUG("cc2420: recv: RSSI is %i\n", (int)rssi);
-
         /* fetch and check if CRC_OK bit (MSB) is set */
         cc2420_fifo_read(dev, &crc_corr, 1);
-        if (!(crc_corr & 0x80)) {
+        if (!(crc_corr & CC2420_CRCCOR_CRC_MASK)) {
```

Comment:
Think this and related changes should go in a separate commit.

Reply:
Good point.